### PR TITLE
Allow switching GPT model in the middle of a conversation

### PR DESCRIPTION
**Pull request by jiayao**
https://github.com/mckaywrigley/chatbot-ui/issues/155

I think it would be useful to combine the speed and cost-effectiveness of gpt-3.5-turbo and the power of gpt-4 in some conversations.

### CODE REVIEW

**Comment by mckaywrigley** [comment]
I really like this idea, though I think the UI needs a bit of cleaning up.

Maybe click on a setting icon in the top bar where the model name displays and it will show this?

**Comment by saliksik** [comment]
I think this button shouldn't be scattered throughout the chat area and should be hidden instead.

**Comment by jiayao** [comment]
Thanks for the feedback, now it's hidden behind a settings icon next to the model name.

File: components/Chat/Chat.tsx [hunk 1]
```diff
@@ -152,9 +158,21 @@ export const Chat: FC<Props> = ({
               </>
             ) : (
               <>
-                <div className="flex justify-center border border-b-neutral-300 bg-neutral-100 py-2 text-sm text-neutral-500 dark:border-none dark:bg-[#444654] dark:text-neutral-200">
-                  {t('Model')}: {conversation.model.name}
+              <div className="flex justify-center border border-b-neutral-300 bg-neutral-100 py-2 text-sm text-neutral-500 dark:border-none dark:bg-[#444654] dark:text-neutral-200">
+                {t('Model')}: {conversation.model.name}
+                <IconSettings className="ml-2" onClick={handleSettings} size={21}/>
```
Comment (mckaywrigley) [inline components/Chat/Chat.tsx:163]: `                  <IconSettings
                    className="ml-2 cursor-pointer hover:opacity-50"
                    onClick={handleSettings}
                    size={18}
                  />`
                  
                  If you could just make this change I'll merge :)

**Pull request merged**